common/components/timeline/macro.njk

```diff
@@ -0,0 +1,27 @@
+{% macro appTimeline(params) %}
+  {%- include "./template.njk" -%}
+{% endmacro %}
+
+{% macro appTimelineItemHeading(params, item) %}
```

Comment:
Is there a reason for needing to create all these separate macros, and to then need to import them all globally in `base.njk`?

As far as I can tell they're only going to be used within `template.njk` in this component?

Reply:
Macros that are not meant to be available globally have been zapped from `base.njk`